cogs/playerstats.py

```diff
@@ -69,7 +69,20 @@ async def assists(self, ctx, competition: str = 'pl'):
         competition_name = self.COMPETITION_TRANSLATIONS[competition.lower()]
 
         await ctx.send('```' + getAssists(competition_name) + '```')
-
+    
+    @commands.command(
+        name="injuries",
+        help="Get the list of injuries and details about them")
+    async def injuries(self, ctx, team: str = 'Arsenal'):
```

Comment:
Why are we taking in a team if the only team it works for is Arsenal?

If the future plan is to allow other teams then imo it should throw an error for now when other teams are tried

Reply:
Sure, I can add an error.